lib/copy_clover_offset.cu

```diff
@@ -1,6 +1,6 @@
 #include <clover_field_order.h>
 #include <instantiate.h>
-#include <kernels/copy_field_offset.cuh>
+#include "copy_field_offset.hpp"
```

Comment:
DO we use any convention for using "header" or <header> ?

Reply:
We have never adopted any convention.  Doing so, would be a global edit now.  I've generally always used `< >` for no particular reason.

Reply:
Guess there is a tool that could do it. But as with clang-format we should delay that. Just noticed that < > changed to " " and thought it might have been intentional.

Reply:
Was likely just a copy-paste change, and nothing of consequence.  Consistent use of header includes could easily be a follow on PR (that comes soon after GK).  Marking resolved for now.